Directory.Packages.props

```diff
@@ -7,6 +7,7 @@
     <PackageVersion Include="Azure.AI.FormRecognizer" Version="4.1.0" />
     <PackageVersion Include="Azure.Extensions.AspNetCore.Configuration.Secrets" Version="1.3.1" />
     <PackageVersion Include="Azure.Identity" Version="1.11.3" />
+    <PackageVersion Include="Azure.AI.OpenAI" Version="1.0.0-beta.17" />
```

Comment:
do you know when this package should go GA?

Reply:
Absolutely no idea, but actually Kernel Memory depends on Semantic Kernel that uses the same package. It is in beta since forever and I really does not know why :(.

Alternatively there is a library maintained by @marcominerva , that is active on Kernel Memory too that fully supports all API. That library is actively maintained by Marco that uses it in production and can be a viable alternative.

Reply:
then perhaps let's link only the SK library, and this will come via SK, using the same version used by SK?

Reply:
It could be a solution, I'll check if it is feasible.